dev/com.ibm.ws.concurrent.persistent_fat_delayexec/test-applications/DelayExecution/resources/WEB-INF/web.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file can be removed, a default web.xml is assumed for servlet-3.0 and newer (I think, for sure 3.1 and newer)